### ISSUE TITLE
Feedback 24 May

### ISSUE THREAD

**Issue by adeldaoud** [AIandGlobalDevelopmentLab/eo-poverty-review#3]
@ksakamoto09 I have the following major comments and some minor in text.

- [ ] 1.	The Review results, qualitatively, is not coming through. One reason is that we have yet not developed a proper categorization. Also, we should list our findings in the appendix at least, with their categories. 
- [ ] 2.	The Schema (or categorization) that order the review is still missing. Again, the Lundberg et al paper is what we need. 
- [x] 3.	Not sure we need the tables in the appendix. They are just a list of keywords. I suggest removing them. 
- [x] 4.	No clear connection between Motivation and Introduction and the section that follows (Classical Geography). 
- [x] 5.	The difference between Section 4 and 5 needs clarification. Both are about EO and so the reader will likely confuse them. My recommendation is to have a Background section where we talk about spatial stats in one subsection and then a subsection of recent trends of using EO for measurement but not causality. Then we introduce our Review in section 5 where we discuss the finding of the literature review and the categorization. Here, we define estimands usual estimands (e.g., from ATE, to CATE, to some local average treatments, geo-RDD style), assumptions (e.g., relaxation of SUTVA).
- [x] 6.	When the reader starts reading section 5, it is actually unclear that they are reading the results of the review. This section reads like the rest. That is not what we want. 
- [x] 7.	Figure 3, for EO-ML causal workflow would be an important contribution. 
- [x] 8.	Bake together Conclusion with Discussion. 
- [x] 9. I am not sure Figure 1 adds much right now. It takes a lot of space with not much added value. 

Can you also address previous feedback #2

**Comment by ksakamoto09** [comment]
3 We can talk about it but we think it might be imporant information for some.

4 We changed the transition is in section 3

5 We fixed the subsection titles to make it more clear what the sections are especially between 4 and 5. 

6 We moved the flowchart from section 5 to the discussion/conclusion part. 

8 we think that the conclusion reads better separate and that it's good to keep separate from discussion.

9 figure 1 is good we think for the motivation.
